.github/workflows/deploy.yml

```diff
@@ -96,6 +96,12 @@ jobs:
             echo "DEPLOY_URL=https://apply-review-$PR_NUMBER.london.cloudapps.digital" >> $GITHUB_ENV
           else
             echo "DEPLOY_ENV=${{ matrix.environment }}" >> $GITHUB_ENV
+            case ${{ matrix.environment }} in
```

Comment:
We could avoid hardcoding the environments by reading from the terraform config. For example:
```shell
hostname=$(jq -r '.service_gov_uk_host_names[0]' ${tf_vars_file})
if [[ $hostname != null ]]; then
  echo "URL=${hostname}.apply-for-teacher-training.service.gov.uk" >> $GITHUB_ENV
else
  echo "URL=apply-${{ matrix.environment }}.london.cloudapps.digital" >> $GITHUB_ENV
fi
```

Reply:
Thanks!